gradle-android-command-plugin/src/test/groovy/com/novoda/gradle/command/AndroidCommandPluginExtensionTest.groovy

```diff
@@ -25,6 +25,11 @@ class AndroidCommandPluginExtensionTest extends GroovyTestCase {
         assert extension.getSeed() == null
     }
 
+    void testDefaultCategories() {
+        def extension = createExtension()
+        assert extension.getCategories() == AndroidCommandPluginExtension.CATEGORIES_DEFAULT
```

Comment:
I see, I still find it a bit odd to pass an empty string when no categories are set though. 
Have to check why `categories` can't just be an empty list when not set otherwise.